protos/vehicle_data.proto

```diff
@@ -114,6 +114,18 @@ enum Field {
   TractorParkBrakeStatus = 104;
   TrailerParkBrakeStatus = 105;
   BrakePedalPos = 106;
+  RouteLastUpdated = 107;
+  RouteLine = 108;
+  MilesToArrival = 109;
+  MinutesToArrival = 110;
+  OriginLocation = 111;
+  DestinationLocation = 112;
+  CarType = 113;
+  Trim = 114;
+  ExteriorColor = 115;
+  RoofColor = 116;
+  ChargePort = 117;
+  ChargePortLatch = 118;
```

Comment:
can you run https://github.com/teslamotors/fleet-telemetry/blob/main/Makefile#L65 to generate python files as well?

Reply:
Good call, done!

Reply:
can you update readme as well?
https://github.com/teslamotors/fleet-telemetry/blob/main/README.md#protos

Reply:
Ah that's where I saw it; done.